docs/shared/react-standalone-tutorial/4-workspace-optimization.md

```diff
@@ -0,0 +1,255 @@
+# React Tutorial - Part 4: Workspace Optimization
```

Comment:
I think this is backwards.

Dependencies between tasks should be covered first, then caching, then affected. The existing tutorials do it incorrectly imo.

The whole affected thing is a thin layer around run-many. It's not really a super core feature, and it has limited utility in the standalone case anyway.

Reply:
I think the title "React Tutorial - Part 4: Workspace Optimization" doesn't work. We aren't really optimizing anything. The whole section is about running tasks.